Upstream-Status: Pending

insserv (1.14.0-5.1) UNRELEASED; urgency=low

  * Non-maintainer upload.
  * moving definition of "extension" in listing.h outside of 
    an if loop with condition related to __STDC_VERSION__.
  * I observed that this gets build with gcc-4.9.2, as for gcc 
    __STDC_VERSION__ is not defined.
  * I got these error while using clang to build this package.
  * I tested it with clang-3.5(__STDC_VERSION__ 199901L) and
    clang 3.7(__STDC_VERSION__ 201112L) and getting same error 
    described in in Bug #777914.

 -- Mohit Bhakkad <mohit.bhakkad@gmail.com>  Sat, 04 Apr 2015 19:14:48 +0000

 insserv (1.14.0-5) unstable; urgency=low
 
   * Add +mountall-bootclean to $local_fs virtual facility definition in
Index: insserv-1.14.0/listing.h
===================================================================
--- insserv-1.14.0.orig/listing.h
+++ insserv-1.14.0/listing.h
@@ -47,9 +47,9 @@ typedef unsigned int uint;
 # ifndef  asm
 #  define asm			__asm__
 # endif
-# ifndef  extension
-#  define extension		__extension__
-# endif
+#endif
+#ifndef  extension
+# define extension             __extension__
 #endif
 #ifndef  attribute
 # define attribute(attr)	__attribute__(attr)
